controller/space.go

```diff
@@ -143,6 +146,15 @@ func (c *SpaceController) Create(ctx *app.CreateSpaceContext) error {
 		return err
 	})
 	if err != nil {
+		// Rool back space and space resource creation
+		c.rollBackSpaceCreation(ctx, spaceID)
+		remoteErr := c.resourceManager.DeleteSpace(ctx, ctx.RequestData, spaceID.String())
```

Comment:
goasupport.ForwardContextRequestID(ctx) ?

Reply:
goasupport.ForwardContextRequestID(ctx) is used inside resourceManager.Create/DeleteSpace()